Latest Docker image broken

Running `docker run -d -p 3001:3001 --name mcp-inspector mcpjam/mcp-inspector:latest` gets the image, but it can't start:

```
2025-09-15 09:31:06.051 | node:internal/modules/cjs/loader:1210
2025-09-15 09:31:06.051 |   throw err;
2025-09-15 09:31:06.051 |   ^
2025-09-15 09:31:06.051 | 
2025-09-15 09:31:06.051 | Error: Cannot find module '/app/dist/server/index.cjs'
2025-09-15 09:31:06.051 |     at Module._resolveFilename (node:internal/modules/cjs/loader:1207:15)
2025-09-15 09:31:06.051 |     at Module._load (node:internal/modules/cjs/loader:1038:27)
2025-09-15 09:31:06.051 |     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)
2025-09-15 09:31:06.051 |     at node:internal/main/run_main_module:28:49 {
2025-09-15 09:31:06.051 |   code: 'MODULE_NOT_FOUND',
2025-09-15 09:31:06.051 |   requireStack: []
2025-09-15 09:31:06.051 | }
2025-09-15 09:31:06.051 | 
2025-09-15 09:31:06.051 | Node.js v20.19.5
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -8,16 +8,17 @@ COPY package.json package-lock.json ./
 COPY client/package*.json ./client/
 COPY server/package*.json ./server/
 RUN npm install --include=dev
-RUN cd client && npm install --include=dev  
+RUN cd client && npm install --include=dev
 RUN cd server && npm install --include=dev
 
 # Stage 2: Build client
 FROM deps-base AS client-builder
 COPY shared/ ./shared/
 COPY client/ ./client/
+COPY .env.production ./
 RUN cd client && npm run build
 
-# Stage 3: Build server  
+# Stage 3: Build server
 FROM deps-base AS server-builder
 COPY shared/ ./shared/
 COPY server/ ./server/
@@ -37,8 +38,14 @@ WORKDIR /app
 COPY --from=client-builder /app/dist/client ./dist/client
 COPY --from=server-builder /app/dist/server ./dist/server
 
-# Copy server package.json and install production dependencies at root
-COPY --from=server-builder /app/server/package.json ./package.json
+# Copy public assets (logos, etc.) to be served at root level
+COPY --from=client-builder /app/client/public ./public
+
+# Copy package.json files for dependencies
+COPY --from=deps-base /app/package.json ./package.json
+COPY --from=deps-base /app/server/package.json ./server/package.json
+
+# Install production dependencies from root package.json (includes external packages like fix-path)
 RUN npm install --production
 
 # Copy shared types
@@ -58,6 +65,10 @@ USER mcpjam
 # Expose port
 EXPOSE 3001
 
+# Set environment variables
+ENV PORT=3001
+ENV NODE_ENV=production
+
 # Health check
 HEALTHCHECK --interval=30s --timeout=3s --start-period=5s --retries=3 \
     CMD node -e "require('http').request('http://localhost:3001/health', (res) => process.exit(res.statusCode === 200 ? 0 : 1)).end()"
@@ -66,4 +77,4 @@ HEALTHCHECK --interval=30s --timeout=3s --start-period=5s --retries=3 \
 ENTRYPOINT ["dumb-init", "--"]
 
 # Start the application with production environment
-CMD ["sh", "-c", "NODE_ENV=production node dist/server/index.cjs"] 
+CMD ["sh", "-c", "NODE_ENV=production node dist/server/index.js"]

--- a/client/src/components/ui/empty-state.tsx
+++ b/client/src/components/ui/empty-state.tsx
@@ -21,9 +21,7 @@ export function EmptyState({
         <div className="w-16 h-16 bg-muted rounded-full flex items-center justify-center mx-auto mb-4">
           <Icon className="h-8 w-8 text-muted-foreground" />
         </div>
-        <h3 className="text-lg font-semibold text-foreground mb-2">
-          {title}
-        </h3>
+        <h3 className="text-lg font-semibold text-foreground mb-2">{title}</h3>
         <p className="text-sm text-muted-foreground mb-4 sm:whitespace-nowrap">
           {description}
         </p>

--- a/server/index.ts
+++ b/server/index.ts
@@ -162,6 +162,13 @@ app.get("/api/mcp-cli-config", (c) => {
 
 // Static file serving (for production)
 if (process.env.NODE_ENV === "production") {
+  // Serve public assets (logos, etc.) at root level
+  app.use("/*.png", serveStatic({ root: "./public" }));
+  app.use("/*.svg", serveStatic({ root: "./public" }));
+  app.use("/*.jpg", serveStatic({ root: "./public" }));
+  app.use("/*.jpeg", serveStatic({ root: "./public" }));
+  app.use("/*.ico", serveStatic({ root: "./public" }));
+
   // Serve static assets (JS, CSS, images, etc.)
   app.use("/*", serveStatic({ root: "./dist/client" }));
 

